matroska_schema_section_header.md

```diff
@@ -2,39 +2,6 @@
 
 This specification includes an `EBML Schema` which defines the Elements and structure of Matroska as an EBML Document Type. The EBML Schema defines every valid Matroska element in a manner defined by the EBML specification.
 
-For convenience the section of the EBML specification that defines EBML Schema Element Attributes is restated here:
```

Comment:
Seems fine to remove this section looks like an older snapshot of the attribute list from the ebml spec.